[2023-03-13T19:30:43.948+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: Spotify_Data_ETL.TransformData manual__2023-03-13T17:35:51.683738+00:00 [queued]>
[2023-03-13T19:30:43.954+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: Spotify_Data_ETL.TransformData manual__2023-03-13T17:35:51.683738+00:00 [queued]>
[2023-03-13T19:30:43.954+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2023-03-13T19:30:43.955+0000] {taskinstance.py:1280} INFO - Starting attempt 18 of 18
[2023-03-13T19:30:43.955+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2023-03-13T19:30:43.963+0000] {taskinstance.py:1300} INFO - Executing <Task(PythonOperator): TransformData> on 2023-03-13 17:35:51.683738+00:00
[2023-03-13T19:30:43.967+0000] {standard_task_runner.py:55} INFO - Started process 11637 to run task
[2023-03-13T19:30:43.971+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'Spotify_Data_ETL', 'TransformData', 'manual__2023-03-13T17:35:51.683738+00:00', '--job-id', '281', '--raw', '--subdir', 'DAGS_FOLDER/spotifydag.py', '--cfg-path', '/tmp/tmpqcjsxlcr']
[2023-03-13T19:30:43.972+0000] {standard_task_runner.py:83} INFO - Job 281: Subtask TransformData
[2023-03-13T19:30:44.021+0000] {task_command.py:388} INFO - Running <TaskInstance: Spotify_Data_ETL.TransformData manual__2023-03-13T17:35:51.683738+00:00 [running]> on host 649b70e6a8b0
[2023-03-13T19:30:44.066+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Ananta Moharana
AIRFLOW_CTX_DAG_ID=Spotify_Data_ETL
AIRFLOW_CTX_TASK_ID=TransformData
AIRFLOW_CTX_EXECUTION_DATE=2023-03-13T17:35:51.683738+00:00
AIRFLOW_CTX_TRY_NUMBER=18
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-03-13T17:35:51.683738+00:00
[2023-03-13T19:30:44.103+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/pandas/util/_decorators.py:311: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  return func(*args, **kwargs)

[2023-03-13T19:30:44.106+0000] {logging_mixin.py:137} INFO - Index(['song_name', 'is_explicit', 'artist_name', 'artist_id'], dtype='object')
[2023-03-13T19:30:44.107+0000] {logging_mixin.py:137} INFO - False
[2023-03-13T19:30:44.108+0000] {logging_mixin.py:137} INFO - **************************
[2023-03-13T19:30:44.108+0000] {logging_mixin.py:137} INFO - Index(['song_name', 'track_id', 'is_explicit', 'artist_name', 'artist_id',
       'acousticness', 'danceability', 'duration_ms', 'energy',
       'instrumentalness', 'key', 'liveness', 'loudness', 'speechiness',
       'tempo', 'valence'],
      dtype='object')
[2023-03-13T19:30:44.109+0000] {logging_mixin.py:137} INFO - False
[2023-03-13T19:30:44.110+0000] {logging_mixin.py:137} INFO - **************************
[2023-03-13T19:30:44.110+0000] {taskinstance.py:1768} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 175, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 192, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/src/transform.py", line 60, in transform_data
    print(song_arist_bridge.columns)
AttributeError: 'NoneType' object has no attribute 'columns'
[2023-03-13T19:30:44.117+0000] {taskinstance.py:1323} INFO - Marking task as FAILED. dag_id=Spotify_Data_ETL, task_id=TransformData, execution_date=20230313T173551, start_date=20230313T193043, end_date=20230313T193044
[2023-03-13T19:30:44.123+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 281 for task TransformData ('NoneType' object has no attribute 'columns'; 11637)
[2023-03-13T19:30:44.142+0000] {local_task_job.py:208} INFO - Task exited with return code 1
[2023-03-13T19:30:44.155+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
